### PR TITLE
[PHP-web] Add functionality to create alphabetic filter. Create test.…

### PR DESCRIPTION
**Pull request by alex-dyak**
… Develop filter, sorting and pagination functionality

### CODE REVIEW

File: src/web/index.php [hunk 1]
```diff
@@ -53,10 +130,19 @@
         Filter by first letter:
 
         <?php foreach (getUniqueFirstLetters(require './airports.php') as $letter): ?>
-            <a href="#"><?= $letter ?></a>
+            <?php
+            if ($query_string) {
+                $href = str_contains($query_string, 'filter_by_first_letter')
+                    ? $_SERVER['DOCUMENT_URI'] . '?' . substr_replace($query_string, $letter, -1, 1)
```
Comment (e-ZAV) [inline src/web/index.php]: $_SERVER['DOCUMENT_URI'] isn't a standard PHP variable. on my server it doesn't work. I had to make a hotfix to test the code locally.

File: src/web/index.php [hunk 2]
```diff
@@ -3,26 +3,103 @@
...
+// For unfiltered pages airports per page is 20. For filtered is 5.
+$offset = (!isset($_GET['filter_by_first_letter']) && !isset($_GET['filter_by_state'])) ? 20 : 5;
+
 // Filtering
 /**
  * Here you need to check $_GET request if it has any filtering
  * and apply filtering by First Airport Name Letter and/or Airport State
  * (see Filtering tasks 1 and 2 below)
  */
+// Filtering tasks 1.
```
Comment (e-ZAV) [inline src/web/index.php]: How safe is it to get values from $_GET without validation? Vulnerability issue.

**Comment by e-ZAV** [comment]
All the logic should be placed in functions.php

File: src/web/index.php (hunk 1 above)
Comment (alex-dyak) [reply src/web/index.php]: Changed $_SERVER['DOCUMENT_URI'] to $_SERVER['SCRIPT_NAME']

File: src/web/index.php (hunk 2 above)
Comment (alex-dyak) [reply src/web/index.php]: Added $_GET validation function

**Review by e-ZAV — APPROVED** [review]

**Review by e-ZAV — APPROVED** [review]

**Pull request merged**